qadence/backends/horqrux/backend.py

```diff
@@ -225,16 +203,24 @@ def _smple(accumulator: ArrayLike, i: int) -> tuple[ArrayLike, None]:
             unhorqify_state=False,
         )
         probs = jnp.abs(jnp.float_power(wf, 2.0)).ravel()
-        samples = [
-            _sample(
-                _probs=probs,
-                n_shots=n_shots,
-                endianness=endianness,
-                n_qubits=circuit.abstract.n_qubits,
-            ),
-        ]
+        key = jax.random.PRNGKey(0)
```

Comment:
is this intentionally left in?

Reply:
@awennersteen it is used in 211 ?

Reply:
Spoke with Dominik and apparently this is how it's done in Jax. 
See e.g.
https://jax.readthedocs.io/en/latest/random-numbers.html

I'll approve but expecting a commit with a comment on this before merging for future reference.

Reply:
Yeah, Dominik explained @Roland-djee 

I think this should have a quick comment left in though for future reference for those not familiar with Jax.

Reply:
Agree.

Reply:
agreed, added